junit-reporting-handlebars/src/main/java/com/github/bogdanlivadariu/reporting/junit/xml/models/TestSuitesModel.java

```diff
@@ -0,0 +1,102 @@
+package com.github.bogdanlivadariu.reporting.junit.xml.models;
+
+import java.util.List;
+import java.util.UUID;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlElementWrapper;
+import javax.xml.bind.annotation.XmlRootElement;
+
+import com.github.bogdanlivadariu.reporting.junit.helpers.Constants;
+
+@XmlRootElement(name = "testsuites")
+@XmlAccessorType(XmlAccessType.FIELD)
+public class TestSuitesModel {
+    @XmlAttribute
+    private String failures;
+
+    @XmlAttribute
+    private String time;
+
+    @XmlAttribute
+    private String errors;
+
+    @XmlAttribute
+    private String tests;
+
+    @XmlAttribute
+    private String name;
+
+    private String uniqueID;
+
+    private String overallStatus;
+
+    @XmlElement(name = "testsuite")
+    private List<TestSuiteModel> testsuite;
+
+    public void postProcess() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The method postProcess() has an NPath complexity of 894](https://www.codacy.com/app/bogdan-livadariu/bootstraped-multi-test-results-report/file/2685730506/issues/source?bid=3293946&fileBranchId=3336457#l40)